sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -21,9 +21,6 @@
     <PackageReference Include="System.Threading.Tasks.Extensions" />
   </ItemGroup>
 
-  <!-- Import the Azure.Core reference -->
```

Comment:
This shouldn't cause an issue if left in, right?    I'd prefer not to remove it so that we don't lose track.   If necessary, can we comment that out instead?

Reply:
It caused a build error because it conflicts with the project reference that is obtained via Azure.Core.Experimental. Note that this is temporary until I can do a release of Azure.Core.Experimental at which point this will switch back to a package reference.

Reply:
Fair enough.   Would you please be kind enough to just comment this out, then, rather than deleting it?